test/test_integration_variable.py

```diff
@@ -0,0 +1,225 @@
+
+        os.remove(file_name)
+
+    @integration_test
+    def test_batch_existing_and_nonexistent_variable_raises_error(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '-i', 'one/password,unknown'], exit_code=1)
+        self.assertIn("404 Client Error", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_get_short_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '-h'], exit_code=0)
+        self.assertIn("usage: variable", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_get_long_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '--help'], exit_code=0)
+        self.assertIn("usage: variable", output)
+
+    @integration_test
+    def test_subcommand_get_help_returns_help(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['variable', 'get'], exit_code=1)
+        self.assertIn("Error the following arguments are required:", self.capture_stream.getvalue())
+
+    @integration_test
+    def test_subcommand_set_help_returns_help(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['variable', 'set'], exit_code=1)
+        self.assertIn("Error the following arguments are required:", self.capture_stream.getvalue())
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_set_long_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'set', '--help'], exit_code=0)
+        self.assertIn("usage: variable set", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_set_short_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'set', '-h'], exit_code=0)
+        self.assertIn("usage: variable set", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_set_variable_with_values_returns_help(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['variable', 'set', '-i', 'one/password'], exit_code=1)
+        self.assertIn("Error the following arguments are required: value", self.capture_stream.getvalue())
+
+
+    @integration_test
+    def test_cli_can_set_and_get_a_defined_variable(self):
+        Utils.assert_set_and_get(self, self.DEFINED_VARIABLE_ID)
```

Comment:
looks like all get commands using this function. I dont see any value for this test.

Reply:
yeah but I wanted one to clearly do that instead of it being part of the other tests but you are right. removing